Add kinetic Alfven wave dispersion solver (#1665)
PlasmaPy/PlasmaPy: changelog/1665.feature.rst

> Added Kinetic Alfven numerical solver to the dispersion package.

Reviewer: ```suggestion
Added a Kinetic Alfvén numerical solver to `plasmapy.dispersion`.
```
